core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -2887,12 +2887,23 @@ struct ViewValueFunctor<DeviceType, ValueType, false /* is_scalar */> {
   construct_dispatch() {
     ValueType value{};
     if (Impl::is_zero_byte(value)) {
+      uint64_t kpID = 0;
+      if (Kokkos::Profiling::profileLibraryLoaded()) {
+        Kokkos::Profiling::beginParallelFor(
```

Comment:
@DavidPoliakoff should we comment on the use of Kokkos::Profiling::cameCase functions versus Kokkos::Tools::Impl::snake_case ones?

Reply:
A comment might be good, but we made the decision intentionally

Reply:
See [`30f1757` (#4246)](https://github.com/kokkos/kokkos/pull/4246/commits/30f1757c46593d3b7ceffb38048c6eeb8304be43).